scripts/clean-install.sh

```diff
@@ -9,6 +9,7 @@ function cleanup() {
 
 trap cleanup EXIT
 
+rm -rf ./clean || true
```

Comment:
This is sort of an unrelated improvement - triggers-end-to-end-test was  sometimes failing because `./clean` already existed when we went to write it, and it was annoying me while I was debugging this.